src/vs/workbench/api/common/menusExtensionPoint.ts

```diff
@@ -52,6 +52,7 @@ namespace schema {
 			case 'notebook/toolbar': return MenuId.NotebookToolbar;
 			case 'dataExplorer/context': return MenuId.DataExplorerContext;
 			case 'dataExplorer/action': return MenuId.DataExplorerAction;
+			case 'notebooks/notebook/title': return MenuId.NotebookTitle;
```

Comment:
why not just notebooks/title. title refers to the top bar on the view container.